charts/vault/templates/vault-rb.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.plugins.readKuberenetsSecrets.namespaces }}
```

Comment:
I don't think this if is needed, the range should be enough isn't it?

Reply:
Fixed that. Thanks